docs/docs/workflows/sharing/README.md

```diff
@@ -0,0 +1,109 @@
+# Sharing Workflows
+
+::: warning Beta Feature
+
+Sharing individual workflows across Pipedream accounts is in Beta and is subject to change.
+
+If you're trying to migrate all resources from one workspace to another, [please contact us for help](mailto:support@pipedream.com).
+
+:::
+
+You can share your workflows as templates with other Pipedream accounts with a unique shareable link.
+
+Creating a share link for your workflow will allow anyone with the link to create a template version of your workflow in their own Pipedream account. This will allow others to use your workflow with their own Pipedream account and also their own connected accounts.
+
+Here's an example of a workflow that will send you a daily SMS message with today's schedule:
+
+<div class="flex justify-center">
+<img alt="Daily Schedule SMS Reminder workflow" src="https://res.cloudinary.com/pipedreamin/image/upload/v1685116771/docs/docs/share%20workflows/New_Project_6_n63kju.png" />
+</div>
+
+Click the button below or copy and paste the link into your browser to instantly create a new workflow from this template:
+
+<div class="flex justify-between my-3">
+  <a href="https://pipedream.com/new?h=tch_ORVf88" class="cursor-pointer pd-copy-workflow shadow-md">
+    <img width="50" src="https://res.cloudinary.com/pipedreamin/image/upload/v1668570410/pipedream-brand-assets/pipedream-icon-96_dgspgo.png" />
+      Copy this Workflow
+  </a>
+
+  <input type="text" class="font-semibold text-center w-3/5 border-2 focus:border-emerald-600  rounded-sm py-2 underline" value="https://pipedream.com/new?h=tch_ORVf88" />
+</div>
+
+
+The copied workflow will include the same trigger, steps and connected account configuration, but it will have a separate event history and versioning from the original.
+
+## Creating a share link for a workflow
+
+To share a workflow, open the **Builder** for the workflow. Then in the top right menu, select **Create Share Link**.
+
+![Click "Create Share Link" in the workflow's settings within the builder to generate a sharable link](https://res.cloudinary.com/pipedreamin/image/upload/v1685119418/docs/docs/share%20workflows/CleanShot_2023-05-26_at_12.42.22_p4q3dr.png)
+
+Now you can define which prop values should be included in this shareable link.
+
+## Including props
+
+Optionally, you can include the actual individual prop configurations as well. This helps speed up workflow development if the workflow relies on specific prop values to function properly.
+
+You can choose to **Include all** prop values if you'd like, or only select specific props.
```

Comment:
```suggestion
For the daily schedule reminder workflow, we included the props for filtering Google Calendar events, but we did not include the SMS number to send the message to. This is because the end user of this workflow uses their own phone number instead:
```